python/knowledges/batch_upload/upload_missing_files.py

```
@@ -0,0 +1,325 @@
+                    break
+            
+            if not file_found:
+                files_not_found.append(filename)
+                print(f"❌ Not found: {filename}")
+        
+        print("\n" + "=" * 50)
+        print(f"📁 Files ready to upload: {len(files_to_upload)}")
+        print(f"❌ Files not found: {len(files_not_found)}")
+        
+        if files_not_found:
+            print(f"\nFiles not found in directory:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)